### PR TITLE
Filter out test data from analytics

### PR DESCRIPTION
**Pull request by gregdingle**
# Description

It looks like robot data was inserted into our active users by one of our tests because the user names were "Admin User" and the traffic only came from ruby. This protects against that happening in future while preserving the ability to test analytics manually in dev environment. 

# Tests

* rails t

### CODE REVIEW

File: app/lib/metric_util.rb [hunk 1]
```diff
@@ -150,5 +150,10 @@ def https_post(uri, data)
         end
       end
     end
+
+    def a_test?(request = nil)
```
Comment (jshoe) [inline app/lib/metric_util.rb]: nit: maybe `is_test?` or `is_a_test?` for a boolean function? similar to `is_a?` https://apidock.com/ruby/Object/is_a%3F

File: app/lib/metric_util_test.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class HeatmapHelperTest < ActiveSupport::TestCase
```
Comment (jshoe) [inline app/lib/metric_util_test.rb]: I think you forgot to change the name HeatmapHelperTest, but also we should use Rspec instead and have it in the /spec folder

Comment (jshoe) [reply app/lib/metric_util_test.rb]: or at least the /test directory with Minitest

File: app/lib/metric_util.rb (hunk 1 above)
Comment (gregdingle) [reply app/lib/metric_util.rb]: I originally wrote `is_a_test?` but rubocop complained.

File: app/lib/metric_util_test.rb (hunk 2 above)
Comment (gregdingle) [reply app/lib/metric_util_test.rb]: 👍

**Pull request merged**